app/client/cli/debug.go

```diff
@@ -66,17 +70,66 @@ func NewDebugSubCommands() []*cobra.Command {
 	return commands
 }
 
-// NewDebugCommand returns the cobra CLI for the Debug command.
-func NewDebugCommand() *cobra.Command {
+// newDebugUICommand returns the cobra CLI for the Debug UI interface.
+func newDebugUICommand() *cobra.Command {
 	return &cobra.Command{
-		Use:               "debug",
-		Short:             "Debug utility for rapid development",
+		Aliases:           []string{"dui", "d"},
+		Use:               "debugui",
+		Short:             "Debug selection ui for rapid development",
 		Args:              cobra.MaximumNArgs(0),
 		PersistentPreRunE: helpers.P2PDependenciesPreRunE,
 		RunE:              runDebug,
 	}
 }
 
+// newDebugCommand returns the cobra CLI for the Debug command.
+func newDebugCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:               "debug",
+		Short:             "Debug utility for rapid development",
+		Args:              cobra.MaximumNArgs(1),
+		PersistentPreRunE: helpers.P2PDependenciesPreRunE,
+	}
+}
+
+func debugCommands() []*cobra.Command {
+	cmds := []*cobra.Command{
+		{
+			Use:     "TriggerView",
+			Short:   "Trigger the next view in consensus",
+			Long:    "Sends a message to all visible nodes on the network to start the next view (height/step/round) in consensus",
+			Aliases: []string{"triggerView"},
```

Comment:
:100: :pray: love this change!

Is it worth considering making the PascalCase the "canonical" form (in `Use`) and moving the dromedaryCase form to be the alias?

My thinking is that the dromedary form is more conventional, in general.